examples/dem/3d-bunny-drill/bunny-drill-loading.prm

```diff
@@ -8,7 +8,7 @@ set dimension = 3
 #---------------------------------------------------
```

Comment:
Do we really need two parameter file? 

The drilling could be done in the same simulation without the restart...